src/assets/seed.data/role.privileges.json

```diff
@@ -63,7 +68,12 @@
             "Address.GetByPersonId",
             "Address.GetByOrganizationId",
             "Doctor.Search",
-            "Doctor.GetByUserId"
+            "Doctor.GetByUserId",
+            "Biometric.BloodOxygenSaturation.Create",
```

Comment:
Should be 
"Biometric**s**.BloodOxygenSaturation.Create",
"Biometric**s**.BloodOxygenSaturation.GetById",